streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBRangeIterator.java

```diff
@@ -44,17 +44,19 @@
         this.forward = forward;
         this.toInclusive = toInclusive;
         if (forward) {
-            iter.seek(from.get());
-            rawLastKey = to.get();
-            if (rawLastKey == null) {
-                throw new NullPointerException("RocksDBRangeIterator: RawLastKey is null for key " + to);
+            if (from == null) {
+                iter.seekToFirst();
+            } else {
+                iter.seek(from.get());
             }
+            rawLastKey = to == null ? null : to.get();
```

Comment:
Should we still need to have `rawLastKey` null check for the `to.get()` case? 

Same comments to the other similar places.

Reply:
Thanks for the comment @showuon , fixed!